runtime/proto/service/log_replication.proto

```diff
@@ -52,4 +52,23 @@ enum LogReplicationEntryType {
   SNAPSHOT_REPLICATED = 4;
   SNAPSHOT_END = 5;
   SNAPSHOT_TRANSFER_COMPLETE = 6;
+}
+
+message LogReplicationSession {
+  string sinkClusterId = 1;
+  string sourceClusterId = 2;
+  ReplicationSubscriber subscriber = 3;
+}
+
+message ReplicationSubscriber {
+  ReplicationModel model = 1;
+  org.corfudb.runtime.UuidMsg clientId = 2;
```

Comment:
What is the usecase for having clientId in addition to a clientName? Considering the case where the same client jvm died and restarted do we really want to have two sessions for the old and new?

Reply:
I did this at a very early stage thinking that it could be any of the two (consumers preference), actually I should've defined it as 'anyOf', we can decide to support only a string client name and remove the clientId.

Reply:
removed the id field as discussed offline